### PR TITLE
Spelling and Grammar edits

### PR DESCRIPTION
**Pull request by plapczyn**
This pull request fixes a few minor spelling and grammar issues.

### CODE REVIEW

**Comment by jhdalek55** [comment]
I'm a big fan of the Oxford commas as well, as I think they add clarity.
But I will leave this to the group.

Lois

**Comment by iramcdonald** [comment]
Hi,

Besides my personal support for ubiquitous use of Oxford comma
(i.e., explicit series comma), ITU-T, SAE, and TCG all rigorously
enforce the use of the Oxford comma.  It's presence is never
ambiguous.  It's absence is often ambiguous, including in some
humorous examples, such as this one from 2018:

*A lack of an Oxford comma cost dairy $5 million*
CNN

A group of Maine dairy delivery drivers will receive $5 million in a
proposed settlement for unpaid overtime, according to court records filed
on Thursday. Read the full story
<https://apple.news/ASXhSsMHHSmKriVHBZY8c-A>
My wife Nancy (a professional editor and writer for over 40 years) sent
this comment
about that news story:

"Splendid!  It's nice to have a real-world example of the dangers of sloppy
writing.
There is never a downside to using the Oxford (or series) comma; it
provides clarity
and hurts nothing.  Leaving it out is merely a senseless fashion."

Cheers,
- Ira

Ira McDonald (Musician / Software Architect)
Co-Chair - TCG Trusted Mobility Solutions WG
Co-Chair - TCG Metadata Access Protocol SG
Chair - Linux Foundation Open Printing WG
Secretary - IEEE-ISTO Printer Working Group
Co-Chair - IEEE-ISTO PWG Internet Printing Protocol WG
IETF Designated Expert - IPP & Printer MIB
Blue Roof Music / High North Inc
http://sites.google.com/site/blueroofmusic
http://sites.google.com/site/highnorthinc
mailto: blueroofmusic@gmail.com
PO Box 221  Grand Marais, MI 49839  906-494-2434

File: uptane-standard.md [hunk 1]
```diff
@@ -282,7 +282,7 @@ The OEM wants to use delta updates to save over-the-air bytes. The delta images
 
 There are a number of factors that could impede the completion of the above scenarios:
 
-* ECUs may be lacking the necessary resources to function as designated. These resources could include weaknesses, in terms of CPU or RAM, that prevent performance of public key cryptography; a lack of sufficient storage to undo installation of bad software; or an ECU simply may reside on a low-speed network (e.g., LIN).
+* ECUs may be lacking the necessary resources to function as designated. These resources could include weaknesses, in terms of CPU or RAM, that make performance of public key cryptography infeasible; a lack of sufficient storage to undo installation of bad software; or an ECU simply may reside on a low-speed network (e.g., LIN).
```
Comment (patrickvacek) [inline uptane-standard.md]: "render" is an even better word than "make" in this context.

File: uptane-standard.md [hunk 2]
```diff
@@ -419,7 +419,7 @@ A repository's Timestamp role SHALL produce and sign metadata indicating whether
 
 ## Metadata structures {#meta_structures}
 
-Uptane's security guarantees all rely on properly created metadata that follow a designated structure. The Uptane standard **does not** mandate any particular format or encoding for the metadata as a whole. ASN.1 (with any encoding scheme like BER, DER, XER, etc.), JSON, XML, or any other encoding format that is capable of providing the required structure MAY be used.
+Uptane's security guarantees all rely on properly created metadata that follows a designated structure. The Uptane standard **does not** mandate any particular format or encoding for the metadata as a whole. ASN.1 (with any encoding scheme like BER, DER, XER, etc.), JSON, XML, or any other encoding format that is capable of providing the required structure MAY be used.
```
Comment (patrickvacek) [inline uptane-standard.md]: "data" is plural so "follow" is correct. I go back and forth on how I really feel about that (I also often treat it as singular), but I think "follow" sounds better here.

File: uptane-standard.md [hunk 3]
```diff
@@ -498,11 +498,11 @@ The Director repository MAY provide a download URL for the image file. This may
 
 #### Metadata about Delegations {#delegations_meta}
 
-A Targets metadata file on the Image repository (but not the Director repository) MAY delegate signing authority to other entities--for example, it could delegate signing authority for a particular ECU's firmware to that ECU's supplier. A metadata file MAY contain more than one delegation, and MUST keep the delegations in prioritized order.
+A Targets metadata file on the Image repository (but not the Director repository) MAY delegate signing authority to other entities--for example, it could delegate signing authority for a particular ECU's firmware to that ECU's supplier. A metadata file MAY contain more than one delegation and MUST keep the delegations in prioritized order.
 
 A list of delegations MUST provide the following information:
 
-* A list of public keys of all delegatees. Each key should have a unique public key identifier, and a key type.
+* A list of public keys of all delegates. Each key should have a unique public key identifier and a key type.
```
Comment (patrickvacek) [inline uptane-standard.md]: I think "delegatee" is more accurate.

**Comment by iramcdonald** [comment]
Hi,

I agree with all of Patrick's wording suggestions.

Cheers,
- Ira

Ira McDonald (Musician / Software Architect)
Co-Chair - TCG Trusted Mobility Solutions WG
Co-Chair - TCG Metadata Access Protocol SG
Chair - Linux Foundation Open Printing WG
Secretary - IEEE-ISTO Printer Working Group
Co-Chair - IEEE-ISTO PWG Internet Printing Protocol WG
IETF Designated Expert - IPP & Printer MIB
Blue Roof Music / High North Inc
http://sites.google.com/site/blueroofmusic
http://sites.google.com/site/highnorthinc
mailto: blueroofmusic@gmail.com
PO Box 221  Grand Marais, MI 49839  906-494-2434

File: uptane-standard.md (hunk 1 above)
Comment (jhdalek55) [reply uptane-standard.md]: I agree. Render is a better word.

File: uptane-standard.md (hunk 2 above)
Comment (jhdalek55) [reply uptane-standard.md]: I go back and forth on this. You are right that, by grammatical rules, "data" is plural. However, its become common usage, I believe, to treat it in the singular form as well. In this case, I agree that "follow" sounds better.

File: uptane-standard.md (hunk 3 above)
Comment (jhdalek55) [reply uptane-standard.md]: Agreed.

**Comment by jhdalek55** [comment]
I don't know if my review counts, but since these were largely wording issues, I think its counts for this PR. I agree with Ira. This can be merged.

**Comment by iramcdonald** [comment]
Hi,

I agree that "follow" sounds better.

Cheers,
- Ira

PS - Lois - your "vote" counts in reviews and edits - you're the
professional
technical writer - the rest of us are just the peanut gallery...

Ira McDonald (Musician / Software Architect)
Co-Chair - TCG Trusted Mobility Solutions WG
Co-Chair - TCG Metadata Access Protocol SG
Chair - Linux Foundation Open Printing WG
Secretary - IEEE-ISTO Printer Working Group
Co-Chair - IEEE-ISTO PWG Internet Printing Protocol WG
IETF Designated Expert - IPP & Printer MIB
Blue Roof Music / High North Inc
http://sites.google.com/site/blueroofmusic
http://sites.google.com/site/highnorthinc
mailto: blueroofmusic@gmail.com
PO Box 221  Grand Marais, MI 49839  906-494-2434

**Comment by jhdalek55** [comment]
Thanks, Ira. I do my my best!

**Comment by iramcdonald** [comment]
Hi Lois,

My wife Nancy (a professional editor and writer for the past 40+ years)
sends you a gold star for your patient attention to consistency and usage
of the "right" words.  I send you a gold star for your ability to run
efficient
meetings of the diverse Uptane contributors.

Cheers,
- Ira

Ira McDonald (Musician / Software Architect)
Co-Chair - TCG Trusted Mobility Solutions WG
Co-Chair - TCG Metadata Access Protocol SG
Chair - Linux Foundation Open Printing WG
Secretary - IEEE-ISTO Printer Working Group
Co-Chair - IEEE-ISTO PWG Internet Printing Protocol WG
IETF Designated Expert - IPP & Printer MIB
Blue Roof Music / High North Inc
http://sites.google.com/site/blueroofmusic
http://sites.google.com/site/highnorthinc
mailto: blueroofmusic@gmail.com
PO Box 221  Grand Marais, MI 49839  906-494-2434

**Comment by jhdalek55** [comment]
Thanks!

**Comment by jhdalek55** [comment]
I reviewed everything here again, including those pesky commas. It's true that the deleted commas here were not the Oxford comma separating lists. They were commas at the end of clauses. I think that type of comma usage is useful, but not as significant as the old Oxfords.

I think this can be merged. Several people have looked at it and I don't think I see any outstanding corrections. I would still argue that, in line 525, that the phrase should be "backwards compatible" with the 's', because it appears to be more commonly used. But I would not hold up a merge to argue about it.

**Comment by jhdalek55** [comment]
Barring any further objections, can someone with the privilege to do so merge this PR. Thanks.

Lois

**Pull request merged**